packages/cdktf-cli/package.json

```diff
@@ -6,9 +6,11 @@
     "cdktf": "bundle/bin/cdktf"
   },
   "scripts": {
-    "build": "tsc",
+    "compile-build-config": "tsc build.ts",
+    "prebuild": "yarn compile-build-config",
+    "build": "tsc --noEmit",
```

Comment:
Why still invoke this if we also have esbuild? For type checking?

Reply:
Yep! Esbuild doesn't typecheck IIRC